README.md

```diff
@@ -1,55 +1,51 @@
+```bash
+pip install naturf
+```
+### Check out a quickstart tutorial to run `naturf`
 
-## Example Data
+Run `naturf` using the quickstart tutorial: [`naturf` Quickstarter](https://immm-sfa.github.io/naturf/quickstarter.html)
 
-The example data provided with this package includes a shapefile containing building footprints and heights for one 3.2km by 3.2km tile of
-Washington, D.C. (C-5.shp) and a CSV file that lists the index name for each tile in Washington, D.C. (DC_Indices.csv).
+### Getting started
 
-NATURF requires the center latitude and longitude of the study area in order to correctly project the output data,
-but these values are already included in `scripts/workflow.py` for the example data.
+New to `naturf`? Get familiar with what `naturf` is all about in our [Getting Started](https://immm-sfa.github.io/naturf/getting_started.html)
 
-Outputs from the example data should include a binary file named `00065-00096.00065-00096`, an `index` file,
-a CSV file with urban parameter values for each building, two NumPy files, several pickle files that are
-used in the compilation of the binary file, and a directory of TIF rasters that represent each of the
-132 calculated urban parameters.
+### User guide
 
-## Workflow
+Our user guide provides in-depth information on the key concepts of `naturf` with useful background information and explanation. See our [User Guide](https://immm-sfa.github.io/naturf/user_guide.html)
 
-1. Example data is provided in this repository in the `example` directory
-2. Setup paths and settings in `scripts/workflow.py`
-3. Run `scripts/workflow.py` in a Python 3.8 or greater environment
+### Contributing to `naturf`
 
-## Citation
+Whether you find a typo in the documentation, find a bug, or want to develop functionality that you think will make `naturf` more robust, you are welcome to contribute! See our [Contribution Guidelines](https://immm-sfa.github.io/naturf/contributing.html)
 
-> Allen-Dumas, Melissa Ree, Sweet-Breu, Levi, Seals, Matthew, Berres, Andy, Vernon, Chris R., Rexer, Emily, and USDOE Office of Science. NATURF. Computer software. https://www.osti.gov//servlets/purl/1879628. Vers. 0. USDOE Office of Science (SC), Biological and Environmental Research (BER). Earth and Environmental Systems Science Division. 31 Aug. 2022. Web. doi:10.11578/dc.20220803.4.
+### API reference
 
+The reference guide contains a detailed description of the `naturf` API. The reference describes how the methods work and which parameters can be used. It assumes that you have an understanding of the key concepts. See [API Reference](https://immm-sfa.github.io/naturf/modules.html)
 
-# Developer Setup
-To get started on development, install the pre-commit hooks to format code.
+### Pre-commit
 
-```bash
-$ brew install pre-commit
-```
-Then install the hooks within the repo:
```

Comment:
we should probably still include the instructions for how to install pre-commit when developing naturf